docs/jwst/tso_photometry/arguments.rst

```diff
@@ -0,0 +1,10 @@
+Step Arguments
+==============
+
+The tso_photometry step has one step-specific argument:
+
+*  ``--save_catalog``
+
+If ``save_catalog`` is set to True (the default is False),
+the output table of times and count rates will be written to an ecsv file
+with suffix "phot".
```

Comment:
Perhaps highlight that this argument is really only needed when you're running the step standalone, because by default when it's run as part of the calwebb_tso3 pipeline the catalog will always be saved.